python/dlr/dlr_model.py

```diff
@@ -160,15 +166,41 @@ def get_input_names(self):
         """
         return self.input_names
 
-    def get_output_names(self):
-        """
-        Get all output names
+    def has_metadata(self) -> bool:
+        flag = ctypes.c_bool()
+        _check_call(_LIB.GetDLRHasMetadata(byref(self.handle), byref(flag)))
+        return flag.value
+    
+    def _fetch_output_names(self):
+        self.output_names = []
+        if self.has_metadata():
```

Comment:
Some backends/models might give you output names without metadata file (e.g. Tensorflow). We should try to fetch `output_names` and if `GetDLROutputName` throws exception we should leave self.output_names empty.